servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -394,3 +393,15 @@ def _format_query_labels(orig_query_labels):
     for key, val in orig_query_labels.items():
         query_labels[urllib.unquote(key)] = urllib.unquote(val)
     return query_labels
+
+
+def _get_scattered_task_status(metadata):
```

Comment:
This is a nice pure function that seems like a great candidate for a unit test

Reply:
👍 Added